versions.json

```diff
@@ -1,114 +1,114 @@
-  },
-  {
-    "id": "0.37.1",
-    "url": "https://639481c052f12500083fa373--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.36.2",
-    "url": "https://635720a7e34ffc00087a288e--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.35.4",
-    "url": "https://62e4858ae62a5c0009e9e7c5--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.34.2",
-    "url": "https://624793d2f1d02d000925c339--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.33.5",
-    "url": "https://6230a5e98514e70008daf4ca--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.32.1",
-    "url": "https://6148d47b4c8c3d0008ecc1cf--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.31.0",
-    "url": "https://6116c4d420b1d40007ce5fd9--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.30.0",
-    "url": "https://6099b79d6eba2400084bb441--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.29.0",
-    "url": "https://6040dbed0b78130008f102b5--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.28.0",
-    "url": "https://60386a93e4a7010007247f23--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.27.0",
-    "url": "https://602a61e08b3cf200074fa0b5--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.26.0",
-    "url": "https://600860290fee570007d7f660--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.25.1",
-    "url": "https://5fdb2c4892110a00080b0895--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.24.1",
-    "url": "https://5fbd1c1239f44a000736c439--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.23.0",
-    "url": "https://5f96ff4047c5f300070eb8a1--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.22.0",
-    "url": "https://5f776fb812cfa300086de86a--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.21.0",
-    "url": "https://5f355f63718e9200075585e1--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.20.0",
-    "url": "https://5f0630964a7a370007f93dc4--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.19.0",
-    "url": "https://5ef288ca158ebd0008946f4d--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.18.0",
-    "url": "https://5ec04b7f46b9bd000648f8ec--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.17.0",
-    "url": "https://5e9a135cc7c8e90006047bdf--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.16.0",
-    "url": "https://5e97032838070600063141e4--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.15.0",
-    "url": "https://5e5fe7518009de0008f41fff--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.14.0",
-    "url": "https://5e4f1d0714bc4c000ae3282d--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.13.0",
-    "url": "https://5e25d4d1837fae0009b5c4fa--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.12.0",
-    "url": "https://5db1dc978d2a340009a82d64--fundamental-ngx.netlify.app/"
-  },
-  {
-    "id": "0.11.0",
-    "url": "https://5d8a3409acaf8d00070ccd64--fundamental-ngx.netlify.app/"
-  }
-]
\ No newline at end of file
+    {
```

Comment:
why is this file changed

Reply:
Formatting maybe from linting or migrations

Reply:
Can you revert it?

Reply:
Not without bypassing the linting tools we have in place